README.md

```diff
@@ -68,12 +68,35 @@ python -m pip install -r requirements.txt
 
 ## How to Build and Run
 
-Migrate the Django database (only required to run once):
+Before starting the server, ensure that the Django database is correctly set up by performing migrations and creating a superuser for administrative access. Follow these steps:
+
+1. **Prepare Database Migrations**:  
+   Initialize database migrations needed for the Django models:
+```
+python manage.py makemigrations
```

Comment:
The makemigrations command is supposed to be called by the developers and not necessarily by the admins. Let's remove it for now.